frontend/app/components/layouts/public-layout.tsx

```diff
@@ -110,8 +110,11 @@ export function BilingualNotFoundError({ error }: BilingualNotFoundErrorProps) {
           <div className="container flex items-center justify-between gap-6 py-2.5 sm:py-3.5">
             <div property="publisher" typeof="GovernmentOrganization">
               <Link to="https://canada.ca/" property="url">
-                <img className="h-8 w-auto" src="/assets/sig-blk-en.svg" alt={`${en('gcweb:header.govt-of-canada.text')} / ${fr('gcweb:header.govt-of-canada.text')}`} property="logo" width="300" height="28" decoding="async" />
+                <img className="h-8 w-auto" src="/assets/sig-blk-en.svg" alt={t('gcweb:header.govt-of-canada.text')} property="logo" width="300" height="28" decoding="async" />
               </Link>
+              <span className="sr-only">
+                / <span lang={i18n.language === 'en' ? 'fr' : 'en'}>{i18n.getFixedT(i18n.language === 'en' ? 'fr' : 'en')('gcweb:header.govt-of-canada.text')}</span>
```

Comment:
I think it would be easier to read and debug if the logic of  this content would be done at the top. Same at the  other place.

```tsx
import { getAltLanguage } from '~/utils/locale-utils';

const altLanguage = getAltLanguage(i18n.language);
const altLogoContent = <span lang={altLanguage}>{i18n.getFixedT(altLanguage)('gcweb:header.govt-of-canada.text')}</span>;
```